kong/plugins/apifortress/capturer.lua

```diff
@@ -0,0 +1,24 @@
+-- Captures the response payloads as they go through the wire
+
+local _M = {}
+
+local function capture()
+  local chunk, eof = ngx.arg[1], ngx.arg[2]
+  local captured_body = ngx.ctx.captured_body
+  if not captured_body then
+    captured_body = {}
+    ngx.ctx.captured_body = captured_body
+  end
+  if chunk ~= "" then
```

Comment:
Actually I am not sure that check is very relevant. If the chunk is an empty string then that's fine, it'll be an empty string during concatenation :) no?